Please get [arm64-android failure log here](https://artprodwus21.artifacts.visualstudio.com/A6adfb432-9816-4cb2-8190-03885865e0d9/c1ee48cb-0df2-4ab3-8384-b1df5a79fe53/_apis/artifact/cGlwZWxpbmVhcnRpZmFjdDovL3ZjcGtnL3Byb2plY3RJZC9jMWVlNDhjYi0wZGYyLTRhYjMtODM4NC1iMWRmNWE3OWZlNTMvYnVpbGRJZC8xMTE1MjcvYXJ0aWZhY3ROYW1lL2ZhaWx1cmUrbG9ncytmb3IrYXJtNjQtYW5kcm9pZA2/content?format=zip) 
```
CMake Error at scripts/cmake/vcpkg_execute_required_process.cmake:127 (message):
    Command failed: /vcpkg/downloads/tools/ninja/1.10.2-linux/ninja -v
    Working Directory: /mnt/vcpkg-ci/b/opensubdiv/arm64-android-rel/vcpkg-parallel-configure
    Error code: 1
    See logs for more information:
      /mnt/vcpkg-ci/b/opensubdiv/config-arm64-android-dbg-CMakeCache.txt.log
      /mnt/vcpkg-ci/b/opensubdiv/config-arm64-android-rel-CMakeCache.txt.log
      /mnt/vcpkg-ci/b/opensubdiv/config-arm64-android-dbg-CMakeConfigureLog.yaml.log
      /mnt/vcpkg-ci/b/opensubdiv/config-arm64-android-rel-CMakeConfigureLog.yaml.log
      /mnt/vcpkg-ci/b/opensubdiv/config-arm64-android-out.log

Call Stack (most recent call first):
  /mnt/vcpkg-ci/installed/x64-linux/share/vcpkg-cmake/vcpkg_cmake_configure.cmake:269 (vcpkg_execute_required_process)
  ports/opensubdiv/portfile.cmake:83 (vcpkg_cmake_configure)
  scripts/ports.cmake:196 (include)
```
**/mnt/vcpkg-ci/b/opensubdiv/config-arm64-android-out.log:**
```
CMake Error at opensubdiv/osd/CMakeLists.txt:417 (install):
  install FILES given no DESTINATION!

-- Configuring incomplete, errors occurred!
```

--- a/ports/opensubdiv/vcpkg.json
+++ b/ports/opensubdiv/vcpkg.json
@@ -1,11 +1,11 @@
 {
   "name": "opensubdiv",
   "version-semver": "3.5.0",
-  "port-version": 2,
+  "port-version": 3,
   "description": "An Open-Source subdivision surface library.",
   "homepage": "https://github.com/PixarAnimationStudios/OpenSubdiv",
   "license": "Apache-2.0",
-  "supports": "!arm & !uwp",
+  "supports": "!(arm & android) & !uwp",
   "dependencies": [
     {
       "name": "vcpkg-cmake",

--- a/versions/baseline.json
+++ b/versions/baseline.json
@@ -6766,7 +6766,7 @@
     },
     "opensubdiv": {
       "baseline": "3.5.0",
-      "port-version": 2
+      "port-version": 3
     },
     "opentelemetry-cpp": {
       "baseline": "1.18.0",

--- a/versions/o-/opensubdiv.json
+++ b/versions/o-/opensubdiv.json
@@ -1,6 +1,11 @@
 {
   "versions": [
     {
+      "git-tree": "b469c74fb77ea987be2333ae2d66ebba168d672d",
+      "version-semver": "3.5.0",
+      "port-version": 3
+    },
+    {
       "git-tree": "8d12fbded4ab0667d082362ae4b6f57506926c9b",
       "version-semver": "3.5.0",
       "port-version": 2

